pages/static/pages/styles_santa.css

```diff
@@ -0,0 +1,120 @@
+body {
+    background: url('../pages/background.jpg') no-repeat center center fixed;
+    background-size: cover;
+    font-family: Arial, sans-serif;
+    color: #333;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+
+.container {
+    flex: 1;
+    max-width: 800px;
+    margin: 0 auto;
+    padding: 20px;
+    background: rgba(255, 255, 255, 0.9);
+    border-radius: 8px;
+}
+
+h1 {
+    text-align: center;
+    color: #FF4500;
+    margin-bottom: 20px;
+}
+
+h2 {
+    font-size: 28px;
+    color: #2F4F4F;
+    border-bottom: 2px solid #FF4500;
+    padding-bottom: 10px;
+    margin-bottom: 20px;
+}
+
+form {
+    margin-bottom: 30px;
+}
+
+input[type="text"] {
+    padding: 10px;
+    width: calc(100% - 22px);
+    margin-bottom: 10px;
+    border: 1px solid #ddd;
+    border-radius: 4px;
+}
+
+button {
+    padding: 10px 15px;
+    background-color: #FF4500;
+    border: none;
+    color: white;
+    border-radius: 4px;
+    cursor: pointer;
+}
+
+button:hover {
+    background-color: #FF6347;
+}
+
+ul {
+    list-style-type: none;
+    padding: 0;
+}
+
+ul li {
+    padding: 10px;
+    background: #F0F8FF;
+    border-radius: 4px;
+    margin-bottom: 5px;
+}
+
+
+.return_button {
+    display: block;
+    width: 200px;
+    margin: 20px auto;
+    color: white;
+    text-align: center;
+    text-decoration: none;
+    padding: 10px 20px;
+    background-color: #ff4500;
+    border: none;
+    border-radius: 4px;
+    font-size: 16px;
+    cursor: pointer;
+}
+
+.return_button:hover {
+    background-color: #ff4500;
+}
+
+form button[name="reset_game"] {
+    background-color: #FF6347;
+    border: none;
+    color: white;
+    padding: 10px 20px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 10px 0;
+    border-radius: 5px;
+    cursor: pointer;
+}
+
+form button[name="reset_game"]:hover {
+    background-color: #FF4500;
+}
+
+.error-message {
+    color: #ff4d4d; /* Red color for error messages */
+    font-size: 12px; /* Smaller font size */
+    margin-top: 10px; /* Space above the message */
+    margin-bottom: 10px; /* Space below the message */
+    padding: 5px; /* Padding around the message */
+    border: 1px solid #ff4d4d; /* Red border around the message */
+    border-radius: 3px; /* Rounded corners for the border */
+    background-color: #ffe6e6; /* Light red background for visibility */
+}
```

Comment:
the red icon at the end of this file means that you need to add an empty line after line 120